src/transform/src/lib.rs

```diff
@@ -471,9 +471,18 @@ impl Optimizer {
             // - there should be no RelationCSE between this LiteralConstraints and
             //   JoinImplementation, because that could move an IndexedFilter behind a Get.
             // - The last RelationCSE before JoinImplementation should be with inline_mfp = true.
-            // - Currently, JoinImplementation has to be in the same fixpoint loop with
+            // - Currently, JoinImplementation can't be before the fixpoint loop that has
```

Comment:
Nit: can you just say 
> can't be before LiteralLifting
here instead (whether the latter is a stand-alone or a fixpoint loop seems irrelevant).

Reply:
Fixed